examples/charts/file/area.py

```diff
@@ -1,6 +1,9 @@
 from collections import OrderedDict
 
-from bokeh._legacy_charts import Area, show, output_file
+from bokeh.charts import Area, show, vplot, output_file, defaults
+
+defaults.width = 400
+defaults.height = 400
 
 # create some example data
 xyvalues = OrderedDict(
```

Comment:
Agree, we should go through and scrub the language across the examples. I have renamed some, but some of the examples still have the xyvalues in there. Some of the examples use names to differentiate the data as needed if the examples script uses more than one source.